Add template docs (#5669)
quickwit-oss/quickwit: docs/configuration/template-config.md

---

> title: Template configuration

Reviewer: ```suggestion
title: Index template configuration
```

sidebar_position: 7
toc_max_heading_level: 4
---

> This page describes how to configure a template.

Reviewer: ```suggestion
This page describes how to configure an index template.
```

> Templates let you dynamically create indexes according to predefined rules. Templates are used automatically when documents are received on the ingest API for an index that doesn't exist.

Reviewer: ```suggestion
Index templates let you dynamically create indexes according to predefined rules. Templates are used automatically when documents are received on the ingest API for an index that doesn't exist.
```
